### ISSUE TITLE
Final project deadlines

### ISSUE THREAD

**Issue by Usual-Virtue** [ataiya/igd#34]
I was looking to clarify some questions around the final projects deadlines.

The report and video are due Friday night but all code is demoed and submitted on the 11th right?

**Comment by drebain** [comment]
Yes, that is correct. You can continue working on your project until Monday.
